sdk/eventgrid/Azure.Messaging.EventGrid/CHANGELOG.md

```diff
@@ -1,6 +1,16 @@
 # Release History
-## 4.0.0-beta.5 (Unreleased)
+## 4.0.0-beta.5 (2021-02-09)
 
+### Added
```

Comment:
By the [guidelines](https://azure.github.io/azure-sdk/policies_releasenotes.html), this should be `New Features`

Reply:
Turns out that is specifically for release notes. Change log guidelines are here - https://azure.github.io/azure-sdk/policies_releases.html#2-change-logs--migration-guides